### DISCUSSION TITLE
cPanel: Unable to upgrade to 1.25 in GUI

### DISCUSSION THREAD

**Discussion by nicm42** [FreshRSS/FreshRSS#7156] [Q&A] [answered]
I'm trying to upgrade to 1.25. I click on the Start Update button, it changes to say Updating... and then redirects to my home page. If I go back to FreshRSS then it's still on 1.24.3. And there's nothing in the Logs about it. But this all worked fine when I upgraded to 1.24.3.

The server is on 8.4 (but I also tried with 8.1 and that did the same thing). The database is 10.6.20-MariaDB. So they're both above the minimums required.

Any ideas on where to start looking for what might be wrong?

**Comment by Alkarex** [comment]
Hello,
What install method? git or manual file upload (ZIP)?

**Reply by nicm42** [reply]
Neither, it was through cPanel with my web host.

**Comment by math-GH** [comment]
Any log entries?

**Reply by nicm42** [reply]
No. The only log entries are about feed errors.

**Reply by Alkarex** [reply]
What about the logs at your Web server level?

**Reply by nicm42** [reply]
Ah, in the /p folder the error_log says: PHP Warning:  PHP Startup: session.gc_divisor must be greater than 0 in Unknown on line 0

**Reply by Alkarex** [reply]
That does not seem to be related to FreshRSS but due to a bad PHP configuration in your `php.ini`

**Reply by Alkarex** [reply]
But maybe this warning also appeared before your FreshRSS update?
In any case, that would require a bit of debugging. I do not know what cPanel is doing

**Reply by Alkarex** [reply, marked as answer]
If possible, you can also consider updating your setup to use git for updates, which is more reliable (still available through the Web UI) and gives more explanations in case of problems such as access rights:
See https://github.com/FreshRSS/FreshRSS/discussions/6769

**Reply by nicm42** [reply]
The first instance of the warning was when I first tried to update to 1.25.0.

My PHP skills are pretty basic and googling says that no one else has ever had that exact error, but similar errors are related to something in the php.ini which isn't in mine. So I don't know either.

It seems like setting it up with git is going to be the best option.

**Reply by nicm42** [reply]
Setting it up with git worked a treat. We'll see what happens next time there's an update, but since it doesn't include cPanel having done who knows what, it should be ok *fingers crossed*. Thanks for you all your help.
